ReportPortal.Tests/ReportPortal.E2E.UI.Business/Pages/Containers/LaunchContainer.cs

```diff
@@ -0,0 +1,29 @@
+﻿using OpenQA.Selenium;
+
+namespace ReportPortal.E2E.UI.Business.Pages.Containers
+{
+    public class LaunchContainer
+    {
+        private readonly IWebElement _webElement;
+
+        public LaunchContainer(IWebElement webElement)
+        {
+            _webElement = webElement;
+        }
+
+        private const string TotalCountLocator = ".//*[contains(@class,'launchSuiteGrid__total-col')]//a";
+        private const string PassedCountLocator = ".//*[contains(@class,'launchSuiteGrid__passed-col')]//a";
```

Comment:
private should be with _totalCountLocator :)

Reply:
Right =)
![Screenshot_1103](https://github.com/Loki16v/report-portal-tests/assets/60069618/b33da088-6755-47cb-a316-c69a4ca61ca5)
, but not in case variables are const or readonly